docs/playground/DESIGN.md

```diff
@@ -0,0 +1,225 @@
+
+
+# Design
+
+Leveraging the power and scale of Amazon Web Service's **(AWS)** Elastic 
+Kubernetes Service **(EKS)** as well as Route 53, OpenSearch Playground allows 
+any prospective users to experiment with OS and OSD. OpenSearch Playground will 
+be open to all (requiring no sign-ups) as a way to showcase the robust 
+capabilities of the services offered by OS/OSD via a secure https connection.
+
+![diagram_white](https://user-images.githubusercontent.com/73027756/173409473-3cc075f3-8ce6-4eb4-95e4-df6f13eb3899.png)
+
+Creating OpenSearch Playground first requires the modification of the OS and OSD bundle configurations . OpenSearch and OpenSearch Dashboards initially require certain permissions to access features, like viewing the dashboards and interacting with sample data. Thus, the images were configured so that anonymous users could view only the features via the enabling of settings in the `config.yml`, `roles.yml`, `roles_mapping.yml`, and `opensearch_dashboards.yml`. 
+
+Sample change for `config.yml`:
+```
+config:
+  dynamic:
+    http:
+      anonymous_auth_enabled: true
+```
+
+Sample change for `roles.yml`:
+```
+opendistro_security_anonymous:
+  reserved: true
+  cluster_permissions:
+    - 'cluster_monitor'
+    - 'cluster:admin/ingest/pipeline/get'
+    - 'cluster:admin/ingest/processor/grok/get'
+    - 'cluster:admin/opendistro/ad/detector/search'
+    - 'cluster:admin/opendistro/ad/detectors/get'
+    - 'cluster:admin/opendistro/alerting/alerts/get'
+    - 'cluster:admin/opendistro/alerting/destination/email_account/get'
+    - 'cluster:admin/opendistro/alerting/destination/get'
+    - 'cluster:admin/opendistro/alerting/monitor/get'
+    - 'cluster:admin/opendistro/asynchronous_search/get'
+    - 'cluster:admin/opendistro/ism/policy/get'
+    - 'cluster:admin/opendistro/rollup/get'
+    - 'cluster:admin/opendistro/reports/definition/get'
+    - 'cluster:admin/opendistro/reports/instance/get'
+    - 'cluster:admin/opensearch/observability/get'
+    - 'cluster:admin/repository/get'
+    - 'cluster:admin/script/get'
+    - 'cluster:admin/snapshot/get'
+    - 'cluster:monitor/task/get'
+    - 'cluster_composite_ops_ro'
+    - 'cluster:admin/opendistro/ad/result/search'
+    - 'cluster:admin/opendistro/ad/tasks/search'
+    - 'cluster:admin/opendistro/alerting/destination/email_account/search'
+    - 'cluster:admin/opendistro/alerting/destination/email_group/search'
+    - 'cluster:admin/opendistro/alerting/monitor/search'
+    - 'cluster:admin/opendistro/ism/policy/search'
+    - 'cluster:admin/opendistro/rollup/search'
+  index_permissions:
+    - index_patterns:
+        - ".kibana"
+        - ".kibana-6"
+        - ".kibana_*"
+        - ".opensearch_dashboards"
+        - ".opensearch_dashboards-6"
+        - ".opensearch_dashboards_*"
+        - "*"
+      allowed_actions:
+        - 'indices:monitor/settings/get'
+        - 'indices:monitor/stats'
+        - 'indices:data/read*'
+        - 'indices:admin/get'
```

Comment:
where is the sample data indices permission? you need to add all the configuration file to different PR, it is kind of implementation, which will be used for deployment.

Reply:
Yes this design document just outlines my changes to the default configurations. I will make another PR to add config files.

> where is the sample data indices permission?
@Flyingliuhub I'm not sure what you mean by this. I've been able to add sample data from admin and see the visualizations/dashboard without the anonymous user adding the data.

Reply:
after install sample data, it will create couple indices, do you have permission to see the data in the dashboards without add those ? maybe I'm wrong here
              - 'opensearch_dashboards_sample_data_logs'
              - 'opensearch_dashboards_sample_data_flights'
              - 'opensearch_dashboards_sample_data_ecommerce'

Reply:
Could it be perhaps because I used "*" in my index patterns that I was able to access all 3 indices? Is the anonymous user allowed to read all of the indices?

Reply:
I would like to avoid * in the role, it is implement detail, nothing related with design, the design should highlight the change, you don't need to add detail here.

Reply:
Alright, I have removed some implementation details from the doc. Now it just displays a sample change and not the full change.